docs/core-concepts/setup-creds.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 🔐 Saving Credentials
+sidebar_label: Saving Credentials
+sidebar_position: 6
+unlisted: false
+---
+
+You can setup your on prem credentials directly in Python:
+
+```python showLineNumbers
+import fused
+
+fused.options.base_url = "***"
```

Comment:
I had simplified this even more from what it [was originally](https://www.notion.so/fusedio/create-page-to-configure-fused-py-for-onprem-credentials-15a899d3b76380ebafa2d8b95d21b231?pvs=25) so I don't know if you want to give more context for users as to what each line should look like?